compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/ComposeFeatureFlags.desktop.kt

```diff
@@ -88,6 +88,14 @@ internal object ComposeFeatureFlags {
     val useInteropBlending = FeatureFlag {
         System.getProperty("compose.interop.blending").toBoolean()
     }
+
+    /**
+     * Whether to propagate unconsumed mouse wheel events to the parent Swing component.
+     *
+     * This flag will be removed in the future, and the default behavior will correspond to a value
+     * of `true`.
+     */
+    val propagateUnconsumedMouseWheelEvents = FeatureFlag { true }
```

Comment:
How it's supposed to be changed outside? I guess it should be `System.getProperty`, right?